off-chain/test/Main.purs

```diff
@@ -18,5 +19,6 @@ main = do
     Just "preprod" -> PreprodRunner.run
     Just "plutip" -> PlutipRunner.run
     Just "unit" -> Spec.test
+    Just "leader" -> Leader.main
```

Comment:
I think that instead, we should integrate tests, or at least `plutip` and `leader` (the condition for the current plutip test can that it ends with 3 success actions) in a single one, and if we can, integrate more test of this kind.